Gaijinizer patch/Map528.txt

```diff
@@ -3,400 +3,497 @@
 > END STRING
 
 > BEGIN STRING
 行かせないよ
 
 \\V[40]にぃは渡さない。
-> CONTEXT: Map528/events/7/pages/0/102/Dialogue/enemy39 < UNTRANSLATED
+> CONTEXT: Map528/events/7/pages/0/102/Dialogue/enemy39
+I won't let you take him away.
 
+I won't give \\V[40]-nii to you.#Ron 
 > END STRING
 
 > BEGIN STRING
 せっかくお兄ちゃんがいい夢を見ていたのに……
 
 起こしてしまうなんて、悪い人たちですね……
-> CONTEXT: Map528/events/7/pages/0/122/Dialogue/enemy39 < UNTRANSLATED
-
+> CONTEXT: Map528/events/7/pages/0/122/Dialogue/enemy39
+You people are awful for waking him up just as
+he was getting to the good part of his dream.#Ron
 > END STRING
 
 > BEGIN STRING
 でも、最後の味方である白兎も居なくなった……
 
 これで、残るは\\C[10]無力なお前だけ
-> CONTEXT: Map528/events/7/pages/0/126/Dialogue/enemy39 < UNTRANSLATED
-
+> CONTEXT: Map528/events/7/pages/0/126/Dialogue/enemy39
+But now that the last of his allies are gone, all
+that's left is the \\C[10]utterly powerless you\\C[0].#Ron
 > END STRING
 
 > BEGIN STRING
 \\C[10]でも、自分にリスクが無いって分かった上で
 最後の脅威だった２人を連れてきたのは、褒めてあげるよ
 
 \\C[0]ちょっとぐらいなら、いい思いさせてあげてもいいかも？
-> CONTEXT: Map528/events/7/pages/0/130/Dialogue/enemy39 < UNTRANSLATED
-
+> CONTEXT: Map528/events/7/pages/0/130/Dialogue/enemy39
+\\C[10]Still, I'll praise you for bringing the last two
+possible threats to us, knowing there'd be no risk
+to you\\C[0]. So how about I do something nice for you?#RonIffy
 > END STRING
 
 > BEGIN STRING
 そうですね……
 
 \\N[1]お兄ちゃんの友達を創造するときの
 材料にぐらいはしてあげましょう
-> CONTEXT: Map528/events/7/pages/0/135/Dialogue/enemy39 < UNTRANSLATED
+> CONTEXT: Map528/events/7/pages/0/135/Dialogue/enemy39
+Yes...
 
+I'll use you as material when we create all of
+Onii-chan's friends.#RonIffy
 > END STRING
 
 > BEGIN STRING
 \\C[2]私は、全ての世界を喰らい尽くす…
 
 お前の世界も、いずれは……
-> CONTEXT: Map528/events/7/pages/0/140/Dialogue/enemy39 < UNTRANSLATED
+> CONTEXT: Map528/events/7/pages/0/140/Dialogue/enemy39
+\\C[2]I'll devour every single world.
 
+Including yours someday...#Ron
 > END STRING
 
 > BEGIN STRING
 あれれ～？
 
 もしかして、私たちに立ち向かう気？
 勝ち目がないって分かってるのに？
-> CONTEXT: Map528/events/7/pages/0/150/Dialogue/enemy39 < UNTRANSLATED
+> CONTEXT: Map528/events/7/pages/0/150/Dialogue/enemy39
+What's this?
 
+Are you maybe planning on fighting us even though
+you know you can never win?#Ron
 > END STRING
 
 > BEGIN STRING
 \\C[10]マイルームのみんなもいない、チェシャも百鬼も居ない……
 ルカやマリオンたちもここには呼べない……
 
 どうやって私たちに勝つつもりなの？
-> CONTEXT: Map528/events/7/pages/0/155/Dialogue/enemy39 < UNTRANSLATED
-
+> CONTEXT: Map528/events/7/pages/0/155/Dialogue/enemy39
+Just how are you planning on winning? Everyone in
+My Room is gone, Cheshire and Hyakki are both
+gone, and you have no way to summon Luka or
+Marion.#Ron
 > END STRING
 
 > BEGIN STRING
 お前のその\\C[10]『諦めない精神』\\C[0]……
 こちらの世界では、随分と役に立ったよ。
 
 オールドワンが部下を作るのに事欠かない……
-> CONTEXT: Map528/events/7/pages/0/160/Dialogue/enemy39 < UNTRANSLATED
-
+> CONTEXT: Map528/events/7/pages/0/160/Dialogue/enemy39
+Your \\C[10]never-give-up spirit \\C[0]has served us well
+in this world. It's given the Old One all it needs
+to create its minions.#Ron
 > END STRING
```

Comment:
Think we're kinda semi-officially localizing "never-give-up spirit" as "determination".